decidim-comments/app/cells/decidim/comments/comment_thread/show.erb

```diff
@@ -1,3 +1,3 @@
-<div class="comment-thread" role="region">
+<div class="comment-thread" role="region" aria-label="Comment <%= model.id %> by <%= model.author.nickname.gsub("_", " ").capitalize %>">
```

Comment:
I would like to have I18n support here. 
Also, do not use the nickname. Try using `author.full_name`. 
```suggestion
<div class="comment-thread" role="region" aria-label="Comment <%= model.id %> by <%= model.author.nickname.gsub("_", " ").capitalize %>">
```